internal/pkg/cli/svc_deploy.go

```diff
@@ -414,6 +420,13 @@ func (o *deploySvcOpts) stackConfiguration(addonsURL string) (cloudformation.Sta
 	var conf cloudformation.StackConfiguration
 	switch t := mft.(type) {
 	case *manifest.LoadBalancedWebService:
+		if err := o.validateAppVersion(t); err != nil {
+			log.Errorf(`Cannot deploy service %s because the application version is incompatible.
+To upgrade the application, please assume the application admin role and run %s first.
```

Comment:
Should we instruct on how to assume the admin role? (Just a thought, maybe it'd be too didactic)

Reply:
sounds like a good idea!